.github/workflows/ci-image-scanning.yaml

```
@@ -0,0 +1,40 @@
+name: image-scanning
+on:
+  push: 
+jobs:
+  use-trivy-to-scan-image:
+    name: image-scanning
+    if: ${{ github.repository == 'karmada-io/karmada' }}
+    runs-on: ubuntu-20.04

```

```suggestion
    runs-on: ubuntu-22.04
```
We are now using `ubuntu-22.04`.